src/java/org/apache/cassandra/cql3/constraints/ColumnConstraints.java

```diff
@@ -216,20 +224,14 @@ public ColumnConstraints prepare(ColumnIdentifier column)
             if (constraints.isEmpty())
                 return NO_OP;
 
-            for (ColumnConstraint<?> constraint : constraints)
```

Comment:
this is redundant now because a user is not specifying it on creation so there is less room to mess it up